app/src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC } from 'react';
 import { render } from 'react-dom';
 import { HashRouter as Router } from 'react-router-dom';
-import { Header } from '@ts-monorepo/components/Header';
+import { Header } from '@ts-monorepo/components/src/Header';
```

Comment:
we shouldn't have **src** in the import

Reply:
We can import from `@ts-monorepo/components`. This is what I meant when I said we can have it more elegant.

Reply:
Changed it to `import { Header } from '@ts-monorepo/components`